android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/views/SurveyQuestion.kt

```diff
@@ -65,6 +67,7 @@ fun SurveyQuestion(
                     modifier = Modifier
                         .align(Alignment.End)
                         .padding(bottom = dimensions.paddingLargest)
+                        .semantics { this.contentDescription = SurveyNextButton + index }
```

Comment:
Same here, we can omit `this.` right?
```suggestion
                        .semantics { contentDescription = SurveyNextButton + index }
```